helm/happa/values.yaml

```diff
@@ -0,0 +1,16 @@
+name: happa
+namespace: giantswarm
+image:
+  repository: giantswarm/happa
+  tag: [[ .Version ]]
+
+userID: 101
+groupID: 101
+
+Installation:
```

Comment:
this `Installation.x` can be removed from here?

Reply:
Otherwise you'll need to define it for testing which is a bit silly.